src/bindings/c/tests/ov_compiled_model_test.cpp

```diff
@@ -12,15 +12,15 @@ TEST_P(ov_compiled_model, ov_compiled_model_inputs_size) {
     ASSERT_NE(nullptr, core);
 
     ov_model_t* model = nullptr;
-    OV_EXPECT_OK(ov_core_read_model(core, xml, bin, &model));
+    OV_ASSERT_OK(ov_core_read_model(core, xml, bin, &model));
```

Comment:
why do we need assert here? I suppose this is ok if we don't have fatal failure here.

Reply:
If read_model failed, the following calling(depends on ov_model) will be invalid.

Reply:
@riverlijunjie In case of fails in read_model, can we always set `ov_model` to NULL?

Reply:
yes, we have initialized ov_model to be null before call ov_core_read_model(), so if it failed, it is still null.

Reply:
in this case we don't see assert, we should use expect. 
Please, read about different for gtest assert and expect.

Reply:
ok, I follow the gtest convention.

Reply:
Some items about "EXPECT" and "ASSERT" in gtest:

Use ASSERT when the condition must hold - if it doesn't the test stops right there. Use this when the remainder of the test doesn't have semantic meaning without this condition holding.

Use EXPECT when the condition should hold, but in cases where it doesn't we can still get value out of continuing the test. (The test will still ultimately fail at the end, though.)

The rule of thumb is: use EXPECT by default, unless you require something to hold for the remainder of the tests, in which case you should use ASSERT for that particular condition.